<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormShow</class>
 <widget class="QWidget" name="FormShow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
   <item>
    <layout class="QGridLayout" name="formshow_gridLayout">
     <property name="spacing">
      <number>1</number>
     </property>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="frmshowpushButton_8">
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="CustomStyle" stdset="0">
        <string>formshow4</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="frmshowpushButton_9">
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="CustomStyle" stdset="0">
        <string>formshow6</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="frmshowpushButton_10">
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="CustomStyle" stdset="0">
        <string>formshow8</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="frmshowpushButton_11">
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="CustomStyle" stdset="0">
        <string>formshow9</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="frmshowpushButton_12">
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="CustomStyle" stdset="0">
        <string>formshow16</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
